scapy/pipetool.py

```diff
@@ -564,7 +569,7 @@ def _print(self, s):
             self.__w.write(s)
             self.__w.close()
         else:
-            os.write(self.__w, s)
+            os.write(self.__w, s.encode("utf8"))
```

Comment:
Do you need the codec here?

Reply:
Oh! you're right...
Just checked the docs, I didn't know you could set an empty codec :/